gpt_index/readers/download.py

```diff
@@ -0,0 +1,67 @@
+"""Download loader from the Loader Hub."""
+
+import json
+import os
+import subprocess
+import sys
+from importlib import util
+
+from langchain.utilities import RequestsWrapper
```

Comment:
if you want to check the status code (see below), i'd prob just use the raw requests module